### ISSUE TITLE
Questions: how to organize link for communications

### ISSUE THREAD

**Issue by XaoGao** [XaoGao/rubywork_ru#47]
Each user can write emails, skypes, telegram, whatsapp, vider and e.t.c how links for communications(in a resume and a vacancy) how we will do it? Create one model with string or create new columns in user model, how to show correct icons, or create model for each communication?

**Comment by realvint** [comment]
My opinion we need a model Contact. A user can have multiple contacts (use nested attr). The Contact field has a type and value. Type make enum for selector (mail, telegram, vk etc)

**Comment by dasmodal** [comment]
I agree with @realvint. That sound good, because not only User may have contact but Company, Job etc.

Also about #55 and #54. Maybe we can keep the contact and the contact type as one entity?
Type field can be enum at the db level or at the model level.

What do you think about?

**Comment by realvint** [comment]
Additional entity ContactType more flexible. Adding new types through the admin panel will be convenient

**Comment by o-200** [comment]
> Type make enum for selector (mail, telegram, vk etc)

great variant! my opinion is give users to select linkedin, hh, telegram or email. May be make sense to publish github at this section
